lib/basedriver/helpers.js

```diff
@@ -357,6 +357,27 @@ function duplicateKeys (input, firstKey, secondKey) {
   return input;
 }
 
+/**
+ * Takes a desired capability and tries to JSON.parse it as an array,
+ * and either returns the parsed array or a singleton array.
+ *
+ * @param {any} cap A desired capability
```

Comment:
missing throws and return descriptions

`@param` is expected to be `{string|Array<String>}`